doc/specs/vulkan/chapters/devsandqueues.txt

```diff
@@ -560,6 +560,14 @@ include::../api/protos/vkCreateDevice.txt[]
   * pname:pDevice points to a handle in which the created sname:VkDevice is
     returned.
 
+fname:vkCreateDevice verifies that the requested extensions are supported (e.g.
+in implementation or enabled instance layer) and if any requested extension is
+not supported, fname:vkCreateDevice must: return ename:VK_ERROR_EXTENSION_NOT_PRESENT.
+After verifying and enabling the extensions the sname:VkDevice object is created
+and returned to the application.
+If a requested extension is only supported by a layer, the
+layer needs to be specified at flink:vkCreateInstance time.
```

Comment:
should probably match the `vkCreateInstance` wording, i.e. the changed:
"(e.g. in the implementation or in any enabled instance layer)"
and
"[...] for the creation to succeed."